universalClient/config/config.go

```diff
@@ -25,35 +45,110 @@ func validateConfig(cfg *Config) error {
 	}
 
 	// Set defaults for registry config
-	if cfg.ConfigRefreshInterval == 0 {
-		cfg.ConfigRefreshInterval = 10 * time.Second
+	if cfg.ConfigRefreshIntervalSeconds == 0 {
+		cfg.ConfigRefreshIntervalSeconds = 60
 	}
 	if cfg.MaxRetries == 0 {
 		cfg.MaxRetries = 3
 	}
-	if cfg.RetryBackoff == 0 {
-		cfg.RetryBackoff = time.Second
+	if cfg.RetryBackoffSeconds == 0 {
+		cfg.RetryBackoffSeconds = 1
 	}
 
 	// Set defaults for startup config
 	if cfg.InitialFetchRetries == 0 {
 		cfg.InitialFetchRetries = 5
 	}
-	if cfg.InitialFetchTimeout == 0 {
-		cfg.InitialFetchTimeout = 30 * time.Second
+	if cfg.InitialFetchTimeoutSeconds == 0 {
+		cfg.InitialFetchTimeoutSeconds = 30
 	}
 
 	// Validate registry config
 	if len(cfg.PushChainGRPCURLs) == 0 {
-		// Default to localhost:9090 if no URLs provided
-		cfg.PushChainGRPCURLs = []string{"localhost:9090"}
+		// Default to localhost (clean base URL without port) if no URLs provided
+		cfg.PushChainGRPCURLs = []string{"localhost"}
```

Comment:
Rather than hardcoding these values, lets use the defaultConfig values here dynamically